### ISSUE TITLE
Porkbun: IPv6 ignored when IPv4 updated

### ISSUE THREAD

**Issue by usul0184** [ddclient/ddclient#570]
When the code for Porkbun attempts to update the IPv4 address, the IPv6 portion is never reached even when desired.  Whether the IPv4 update succeeds or fails, a `next` statement skips over the IPv6 section in the loop.

**Comment by jortkoopmans** [comment]
Just a quick glance through the protocol code (while not a user of Porkbun myself), indeed it seems impossible to update both ipv4 and ipv6 for a host, whereas either one alone should work.
Problem is that it's impossible to reach the second if in this scenario: `if (is_ipv6($ipv6)`

Essentially:
```pl
foreach my $host (@_) {
  my $ipv4 = delete $config{$host}{'wantipv4'};
  my $ipv6 = delete $config{$host}{'wantipv6'};
  
  if (is_ipv4($ipv4)) {
    do_stuff()
    next #there's several next's here in the control flow
  }
  if (is_ipv6($ipv6){
    cannot_be_reached_when_ipv4_is_also_set()
  }
}
```

Note that there is quite some code duplication in this protocol implementation.

**Comment by jortkoopmans** [comment]
@usul0184 : Can you upgrade to latest release v3.11.1 and verify if your problem still exists?

**Comment by LenardHess** [comment]
Closed due to inactivity. If there is an issue with the fix from #557 please reopen.
